plugins/search/src/plugin.ts

```diff
@@ -60,67 +65,14 @@ export const SearchPage = searchPlugin.provide(
 );
 
 /**
- * @deprecated This component was used for rapid prototyping of the Backstage
- * Search platform. Now that the API has stabilized, you should use the
- * <SearchPage /> component instead. This component will be removed in an
- * upcoming release.
- */
-export const SearchPageNext = searchPlugin.provide(
```

Comment:
It would be worth creating a separate, `minor` changeset just for these `*Next` removals, I think.  With a message like...

> The pre-alpha `<SearchPageNext>`, `<SearchBarNext>`, `etc...` components have been removed.  In the unlikely event you were still using/referencing them, please update to using their non-`*Next` equivalents from either `@backstage/plugin-search-react` or `@backstage/plugin-search`.

...Or something similar.